shell/platform/darwin/ios/rendering_api_selection.mm

```diff
@@ -49,8 +57,15 @@ Class GetCoreAnimationLayerClassForRenderingAPI(IOSRenderingAPI rendering_api) {
       return [CALayer class];
     case IOSRenderingAPI::kOpenGLES:
       return [CAEAGLLayer class];
-#if !TARGET_IPHONE_SIMULATOR
     case IOSRenderingAPI::kMetal:
+#if TARGET_IPHONE_SIMULATOR
+      // This will always be true since we filter out this case when checking if the device
+      // supports metal.
+      if (@available(iOS 13.0, *)) {
+        return [CAMetalLayer class];
+      }
```

Comment:
If this is always true but you have to add the @available for the compiler, add an assertion right after this clause that tears down the process. Just `FML_CHECK(false) << "The reason in the comment"` should be sufficient.

Reply:
The assertion with comment is generally a good idea anywhere else you had to add a check seemingly unnecessarily. That give future readers a clue about why that was added.